src/main/java/com/esri/samples/scene/open_mobile_scene_package/README.md

```diff
@@ -0,0 +1,35 @@
+# Open mobile scene package
+
+Open and display a scene from an offline Mobile Scene Package (.mspk).
+
+![](OpenMobileScenePackageSample.png)
+
+## Use Case
+A .mspk file is an archive containing the data (specifically, basemaps and features) used to display an offline 3D scene.
+
+## How it works
+1. Create a `String` pointing to the samples-data/mspk directory which hosts the .mspk data.
```

Comment:
Once you move those lines into the if/else statement, I don't think you'll need this line. Plus this should be more general:
"Instantiate a `MobileScenePackage` with the path to the .mspk file", "Instantiate a `MobileScenePackage` with the path to the unpacked mspk".